Simulator/Test/test_sun_sensor_math.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Nov 30 17:22:15 2019
+
+"""
+import pytest
+import os, sys, inspect
+import sun_sensor_math as LA
+import numpy as np
+
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+groundtruth_dir = os.path.dirname(currentdir)
+sys.path.insert(0, groundtruth_dir)
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--cmdopt", action="store", default="type1", help="my option: type1 or type2"
+    )
+
+@pytest.fixture
+def cmdopt(request):
+    return request.config.getoption("--cmdopt")
+
+def test_sense2vector_error():
+    with pytest.raises(TypeError):
```

Comment:
I don't think you can put two erroring lines in the same `with pytest.raises` block. (I'm not 100% sure on this so correct me if I'm wrong)

Reply:
I must have misunderstood the examples. I didn't see anything explicitly saying one way or the other, but these can easily be rewritten with a single erroring line.

Reply:
Just checked, you can't put two statements in the same `with pytest.raises`. See the following:

import pytest
import math

```python
def test_raises():
    with pytest.raises(TypeError):
        math.sin([10, 11])
        math.sin(10)
```
You're trying to tests that _both_ statements fail, and this only catches if _any_ of them fail (comment out the first `sin`to see what I mean)

Reply:
@kevincrispie note that you can put multiple tests in the same function, you just can't put multiple error-tests in the same with `pytest.raises` block. See how I updated `test_conversions.py` if that still doesn't make sense. I threw in a couple of error tests.